sources/api/thar-be-settings/src/config.rs

```diff
@@ -148,10 +159,32 @@ impl RenderedConfigFile {
             })?;
         };
 
-        fs::write(&self.path, self.rendered.as_bytes()).context(error::TemplateWriteSnafu {
-            path: &self.path,
-            pathtype: "file",
-        })
+        let mut binding = OpenOptions::new();
+        let options = binding.write(true).create(true).mode(DEFAULT_FILE_MODE);
+
+        // See if this file has a config setting for a specific mode, but don't error if we are
```

Comment:
nit: this comment is out-of-date now that we error out

Reply:
Cleaned up.